app/models/post.rb

```diff
@@ -0,0 +1,2 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] Kindly consider creating associations between models according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
  - two examples of association here would be like 
```
belongs_to :author, class_name: 'User'
has_many :comments
````